### ISSUE TITLE
Is this library maintained?

### ISSUE THREAD

**Issue by algorithmcardboard** [mattn/go-oci8#433]
Basically the title.  I dont see any commits after January of 2021.  Is this library still maintained?

**Comment by mattn** [comment]
Yes. However, it is not aggressive.

**Comment by mattn** [comment]
Yes — the project is active again: CI was migrated to GitHub Actions running the full test suite against Oracle Free 23 (#441), and a series of fixes has been merged recently (#442-#451), including a crash fix in LOB reads, a working database/sql Connector, context-aware Ping, and time.Time OUT binds.
